components/monday/actions/create-board/create-board.mjs

```diff
@@ -0,0 +1,72 @@
+import monday from "../../monday.app.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  key: "monday-create-board",
+  name: "Create Board",
+  description: "Creates a new board. [See the docs here](https://api.developer.monday.com/docs/boards#create-a-board)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    monday,
+    boardName: {
+      propDefinition: [
+        monday,
+        "boardName",
+      ],
+    },
+    boardKind: {
+      propDefinition: [
+        monday,
+        "boardKind",
+      ],
+    },
+    folderId: {
+      propDefinition: [
+        monday,
+        "folderId",
+      ],
+    },
+    workspaceId: {
+      propDefinition: [
+        monday,
+        "workspaceId",
+      ],
+    },
+  },
+  async run({ $ }) {
```

Comment:
I tried adding `templateId` back in and couldn't get it to work with any of the "predefined templates" (the ones you added to templates.mjs). It always returns this error:

`Failed to create board [CreateBoardException]: Failed to create board from predefined template`

However, I did get it to work using the templateId/boardId of a board I had "Saved as template". I think we should leave `templateId` in as an optional integer field, and let the user enter a templateId if they have one.

```suggestion
    },
    templateId: {
       propDefinition: [
          monday,
          "templateId",
       ],
    },
  },
  async run({ $ }) {
```